ignite/cmd/cmd.go

```diff
@@ -78,6 +78,7 @@ To get started, create a blockchain:
 	c.AddCommand(NewDocs())
 	c.AddCommand(NewVersion())
 	c.AddCommand(NewPlugin())
+	c.AddCommand(NewDoctor())
```

Comment:
can we use the variadic method to add all sub commands?

```go
c.AddCommand(
  ...
  NewVersion()
  NewPlugin(),
  NewDoctor(),
  ...
)
 ```

Reply:
That makes the diff less readable, I would rather not do that in this PR.